ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ranger/RangerPolicy.java

```diff
@@ -0,0 +1,1513 @@
+    private static final long serialVersionUID = 1L;
+    private RangerPolicyItemDataMaskInfo dataMaskInfo;
+
+    public RangerDataMaskPolicyItem() {
+      this(null, null, null, null, null, null);
+    }
+
+    public RangerDataMaskPolicyItem(List<RangerPolicyItemAccess> accesses,
+                                    RangerPolicyItemDataMaskInfo dataMaskDetail, List<String> users,
+                                    List<String> groups,
+                                    List<RangerPolicyItemCondition> conditions, Boolean delegateAdmin) {
+      super(accesses, users, groups, conditions, delegateAdmin);
+      setDataMaskInfo(dataMaskDetail);
+    }
+
+    /**
+     * @return the dataMaskInfo
+     */
+    public RangerPolicyItemDataMaskInfo getDataMaskInfo() {
+      return dataMaskInfo;
+    }
+
+    /**
+     * @param dataMaskInfo the dataMaskInfo to set
+     */
+    public void setDataMaskInfo(RangerPolicyItemDataMaskInfo dataMaskInfo) {
+      this.dataMaskInfo = dataMaskInfo == null ? new RangerPolicyItemDataMaskInfo() : dataMaskInfo;
+    }
+
+    @Override
+    public int hashCode() {
+      final int prime = 31;
+      int result = super.hashCode();
+      result = prime * result + ((dataMaskInfo == null) ? 0 : dataMaskInfo.hashCode());
+      return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+      if (!super.equals(obj)) {
+        return false;
+      }
+      if (this == obj) {
+        return true;
+      }
+      if (obj == null) {
+        return false;
+      }
+      if (getClass() != obj.getClass()) {
+        return false;
+      }
+      RangerDataMaskPolicyItem other = (RangerDataMaskPolicyItem) obj;
+      if (dataMaskInfo == null) {
+        if (other.dataMaskInfo != null) {
+          return false;
+        }
+      } else if (!dataMaskInfo.equals(other.dataMaskInfo)) {
+        return false;
+      }
+      return true;
+    }
+
+    @Override
+    public String toString() {
+      StringBuilder sb = new StringBuilder();
+      toString(sb);
+      return sb.toString();
+    }
+
+    public StringBuilder toString(StringBuilder sb) {
+      sb.append("RangerDataMaskPolicyItem={");
+      super.toString(sb);
+      sb.append("dataMaskInfo={");
+      if (dataMaskInfo != null) {
+        dataMaskInfo.toString(sb);
+      }
+      sb.append("} ");
+      sb.append("}");
+      return sb;
+    }
+  }
+
+  /**
+   * RangerRowFilterPolicyItem class.
+   */
+  @JsonAutoDetect(fieldVisibility = Visibility.ANY)
+  @JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
+  @JsonIgnoreProperties(ignoreUnknown = true)
+  @XmlRootElement
+  @XmlAccessorType(XmlAccessType.FIELD)
+  public static class RangerRowFilterPolicyItem extends RangerPolicyItem implements java.io.Serializable {
+    private static final long serialVersionUID = 1L;
+    private RangerPolicyItemRowFilterInfo rowFilterInfo;
+
+    public RangerRowFilterPolicyItem() {
```

Comment:
Unused? Remove it.

Reply:
Needed for deserialize, the default constructor